modules/rosa-policy-change-management.adoc

```diff
@@ -7,53 +7,33 @@
-
-Cluster changes are initiated in one of two ways:
-
-1. A customer initiates changes through self-service capabilities such as cluster deployment, worker node scaling, or cluster deletion.
-2. Red Hat site reliability engineering (SRE) initiates a change through Operator-driven capabilities such as configuration, upgrade, patching, or configuration changes.
-
-Change history is captured in the Cluster History section in {cluster-manager} Overview tab and is available to customers. The change history includes, but is not limited to, logs from the following changes:
-
-- Adding or removing identity providers
-- Adding or removing users to or from the `dedicated-admins` group
-- Scaling the cluster compute nodes
-- Scaling the cluster load balancer
-- Scaling the cluster persistent storage
-- Upgrading the cluster
-
-The SRE-initiated changes that require manual intervention by SRE generally follow this process:
-
-- Preparing for change
-* Change characteristics are identified and a gap analysis is performed against current state.
-* Change steps are documented and validated.
-* A communication plan and schedule are shared with all stakeholders.
-* CI/CD and end-to-end tests are updated to automate change validation.
-* A change request that captures change details is submitted for management approval.
-- Managing change
-* Automated nightly CI/CD jobs pick up the change and run tests.
-* The change is made to integration and stage environments, and manually validated before updating the customer cluster.
-* Major change notifications are sent before and after the event.
-- Reinforcing the change
-* Feedback on the change is collected and analyzed.
-* Potential gaps are diagnosed to understand resistance and automate similar change requests.
-* Corrective actions are implemented.
+This section describes the policies about how cluster and configuration changes, patches, and releases are managed.
 
-[NOTE]
-====
-SRE only uses manual changes as a fallback process because manual intervention is considered to be a failure of change management.
-====
+[id="rosa-policy-customer-initiated-chaneges_{context}"]
+== Customer-initiated changes
+
+You can initiate changes using self-service capabilities such as cluster deployment, worker node scaling, or cluster deletion.
 
-[id="rosa-policy-configuration-management_{context}"]
-== Configuration management
+Change history is captured in the *Cluster History* section in OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
 
-The infrastructure and configuration of the {product-title} environment is managed as code. SRE manages changes to the {product-title} environment using a GitOps workflow and automated CI/CD pipeline.
+* Adding or removing identity providers
+* Adding or removing users to or from the dedicated-admins group
```

Comment:
I think people will understand but maybe:

```suggestion
* Adding users to or removing users from the dedicated-admins group
```

Also should `dedicated-admins` be marked up in any way? (I have no idea)

Reply:
I think so. Good catch!